ts-tests/scripts/smoke-test.ts

```diff
@@ -0,0 +1,287 @@
+import { ApiPromise, Keyring, WsProvider } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { U8aFixed, UInt } from "@polkadot/types/codec";
+import { TypeRegistry } from "@polkadot/types/create";
+import { Text, U128 } from "@polkadot/types/primitive";
+import * as crypto from "@polkadot/util-crypto";
+// Import Web3 from 'web3';
+import { testValidator } from "@polkadot/util-crypto/base32/is";
+import { expect } from "chai";
+
+// Import elliptic crypto
+// import { elliptic } from 'elliptic';
+var elliptic = require("elliptic");
+const ec = new elliptic.ec("secp256k1");
+
+// Import eth lib (wrapper of elliptic lib)
+var Account = require("eth-lib/lib/account");
+var Hash = require("eth-lib/lib/hash");
+
+const privateKey =
+  "0xe82c0c4259710bb0d6cf9f9e8d0ad73419c1278a14d375e5ca691e7618103011";
+
+// Provider is set to localhost for development
+const wsProvider = new WsProvider("ws://localhost:9944");
+
+// Keyring needed to sign using Alice account
+const keyring = new Keyring({ type: "sr25519" });
+
+// Configs of test ropsten account
+const test_eth_address = "[0x4d88dc5d528a33e4b8be579e9476715f60060582]";
+
+const msgPrefix: string = "Link Litentry: ";
+// const msgPrefix: string = "\x19Ethereum Signed Message:\n51Link Litentry: ";
+
+const keyringRopsten = new Keyring({ type: "ecdsa" });
+
+// Setup the API and Alice Account
+async function init() {
+  console.log(
+    `Initiating the API (ignore message "Unable to resolve type B..." and "Unknown types found...")`
+  );
+
+  // Initiate the polkadot API.
+  const api = await ApiPromise.create({
+    provider: wsProvider,
+    types: {
+      // mapping the actual specified address format
+      Address: "AccountId",
```

Comment:
what's the difference between "AccountId" and "MultiAddress"

I use "MultiAddress" in registrar. I tried "AccoundId", but failed.

Reply:
I never used MultiAddress, and this config is from the polkadot js api. Did you use the same polkadot version here?

Reply:
Multi address is a new type of address that supports also say ethereum like address and many others.
Check https://github.com/substrate-developer-hub/substrate-node-template/pull/130

Reply:
We also tried to introduce it in our lates parachain code #6